apisix/cli/ngx_tpl.lua

```diff
@@ -146,6 +146,9 @@ http {
     lua_shared_dict jwks                  1m; # cache for JWKs
     lua_shared_dict introspection        10m; # cache for JWT verification results
 
+    # for authz-keycloak
+    lua_shared_dict access_tokens         1m; # cache for service account access tokens
```

Comment:
Need to add it after:
https://github.com/apache/apisix/blob/bbbdf58d555ee89144b6923e751deabf7e0bbf15/t/APISIX.pm#L248

Reply:
Will do. Can you quickly explain the reason?

Reply:
The configuration used in test is generated from `apisix/t/APISIX.pm` instead of `apisix/cli/ngx_tpl.lua`.

Reply:
Ah, ok.

Reply:
@spacewander Quick question: We also need a shared dict for the discovery documents. I have added that to `apisix/t/APISIX.pm` now as well. But I can't see other dicts that e.g. the `openid-connect` plugin needs in that file. How do the tests then run successfully w/o the dict definitions?